### ISSUE TITLE
Can you share the training log of t2m_trans

### ISSUE THREAD

**Issue by CDLCHOI** [exitudio/MMM#7]
Can you share the training log of t2m_trans?   
I found it difficult to train t2m_trans.

**Comment by CDLCHOI** [comment]
Thanks. 
I load your trans.pth , and finetune with lr=1e-4, at the begining of training, loss≈2.2, acc≈60%。
1. And I keep training, at the end the loss go down to <0.5, acc to 80%+ , did you choose the middle checkpoint as the trans.pth？
2. And I test the loss 0.5 acc 80%+ checkpoint， its FID and R_precision is obviously worse than the trans.pth your offered.  Why does this happen

**Comment by exitudio** [comment]
1. No. It's save from the last epoch.
2. If you intend to train a Transformer, only load the pretrained vqvae.pth model. Perhaps it has been overtrained.

**Comment by CDLCHOI** [comment]
But I think my model is not overtrained.
I train from scratch  again  with your code in https://github.com/exitudio/MMM/issues/6 .
`python3 train_t2m_trans.py  \
    --exp-name validation_train \
    --batch-size 128 \
    --vq-name pretrain \
    --out-dir output/test \
    --total-iter 300000 \
    --lr-scheduler 150000 \
    --dataname t2m \
    --eval-iter 20000`

and this is part of my log(the code didn't eval )
`2024-04-19 09:52:15,403 INFO Train. Iter 10 : Loss. 8.96595,acc:0.58339
2024-04-19 10:13:31,970 INFO Train. Iter 4000 : Loss. 5.02665,acc:10.64897
2024-04-19 10:34:56,329 INFO Train. Iter 8000 : Loss. 3.43890,acc:25.89595
2024-04-19 10:56:53,287 INFO Train. Iter 12000 : Loss. 2.06721,acc:43.09161
2024-04-19 11:46:31,417 INFO Train. Iter 20000 : Loss. 1.05492,acc:66.60687
2024-04-19 12:07:59,340 INFO Train. Iter 24000 : Loss. 1.05645,acc:68.19553
2024-04-19 12:40:16,859 INFO Train. Iter 30000 : Loss. 0.82608,acc:77.09264
2024-04-19 15:22:51,580 INFO Train. Iter 32290 : Loss. 0.75970,acc:69.74814`


and this is part of log finetuning with  "resume_trans": "pretrain/trans.pth",
`    "resume_pth": "./output/vq/vq_name/net_last.pth",
    "resume_trans": "pretrain/trans.pth",
    "root_dist_loss": false,
    "root_loss_no_vel_rot": false,
    "save_iter": 2000,
    "seed": 123,
    "seq_len": 64,
    "stride_t": 2,
    "temporal_complete": 0.0,
    "text": null,
    "total_iter": 30000,
    "traj_supervise": false,
    "vq_act": "relu",
    "vq_dir": "./output/vq/vq_name",
    "vq_name": "output/vq/vq_name",
    "warm_up_iter": 1000,
    "weight_decay": 1e-06,
    "width": 512,
    "xyz_type": "all"
}
2024-04-18 12:05:16,756 INFO Train. Iter 20 : Loss. 2.88080
2024-04-18 12:05:20,199 INFO Train. Iter 40 : Loss. 2.79312
2024-04-18 12:05:23,651 INFO Train. Iter 60 : Loss. 2.79212
2024-04-18 12:08:05,232 INFO Train. Iter 1000 : Loss. 2.09924
2024-04-18 12:10:58,100 INFO Train. Iter 2000 : Loss. 1.39754
2024-04-18 12:16:39,970 INFO Train. Iter 3980 : Loss. 1.06009
2024-04-18 12:16:43,554 INFO Train. Iter 4000 : Loss. 1.02272
2024-04-18 12:22:30,369 INFO Train. Iter 6000 : Loss. 0.94934
2024-04-18 12:28:35,686 INFO Train. Iter 8100 : Loss. 0.77994
2024-04-18 12:35:36,789 INFO Train. Iter 10380 : Loss. 0.64359
2024-04-18 12:35:40,204 INFO Train. Iter 10400 : Loss. 0.63709
2024-04-18 12:43:06,749 INFO Train. Iter 13000 : Loss. 0.47693
2024-04-18 12:45:58,573 INFO Train. Iter 14000 : Loss. 0.44967
`

I'm confused about the 2 results. 
Do you know the reason? Or can you finetuning with trans.pth for a little while with lr=1e-4.

Sorry to bother you again.

**Comment by exitudio** [comment]
Can you try to train longer? It seems like you're using a batch size of 128 with 30,000 epochs.

I use batch size of 512 with 75,000 epochs. I also tried batch size 128 with more epochs, 300,000 epochs (to make total data the same). These 2 settings have similar results.

If you have any further questions, please don't hesitate to ask. I'll do my best to assist you.

**Comment by CDLCHOI** [comment]
Thank you so much. I will try again
